src/mono/browser/runtime/loader/icu.ts

```diff
@@ -66,8 +66,9 @@ export function getIcuResourceName (config: MonoConfig): string | null {
 
         let icuFile = null;
         if (config.globalizationMode === GlobalizationMode.Custom) {
+            // custom ICU file is saved in the resources with fingerprinting and does not require mapping
             if (icuFiles.length === 1) {
```

Comment:
let's fix the condition too. 
```suggestion
            if (icuFiles.length >= 1)
```

Reply:
What's the scenario when there is more then one file?